### PR TITLE
Translation Updates

### PR DESCRIPTION
**Pull request by vetrivelc**

### CODE REVIEW

File: XenAdmin/MainWindow.ja.resx [hunk 1]
```diff
@@ -2445,6 +2445,12 @@
   <data name="DestroyStorageRepositoryToolStripMenuItem.Text" xml:space="preserve">
     <value>破棄(&amp;O)</value>
   </data>
+  <data name="ConvertToThinStorageRepositoryToolStripMenuItem.Size" type="System.Drawing.Size, System.Drawing">
```
Comment (MihaelaStoica) [inline XenAdmin/MainWindow.ja.resx]: These controls have been removed in PR #1728, where we removed all the thin provisioning code. Please make sure that we are not regressing that commit.

File: XenAdmin/Dialogs/DuplicateTemplateNameDialog.ja.resx [hunk 2]
```diff
@@ -253,6 +253,6 @@
     <value>DuplicateTemplateNameDialog</value>
   </data>
   <data name="&gt;&gt;$this.Type" xml:space="preserve">
-    <value>XenAdmin.Dialogs.XenDialogBase, XenCenterMain, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null</value>
+    <value>XenAdmin.Dialogs.XenDialogBase, XenCenter, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null</value>
```
Comment (MihaelaStoica) [inline XenAdmin/Dialogs/DuplicateTemplateNameDialog.ja.resx]: In PR #1803 we changed "XenCenter" to "XenCenterMain" in some of the resource files. Please make sure that we are not regressing that commit.

**Comment by vetrivelc** [comment]
I am working on the fix for commented issues

**Comment by vetrivelc** [comment]
forked source repo got corrupted. hence deleting this MR

**Comment by vetrivelc** [comment]
New PR#https://github.com/xenserver/xenadmin/pull/1827

**Pull request closed without merging**